src/pages/lectures.tsx

```diff
@@ -0,0 +1,131 @@
+import type { HeadFC, PageProps } from "gatsby"
+import * as React from "react"
+import { useTranslation } from "react-i18next"
+import { CommonHead } from "../components/Head/CommonHead"
+import { Layout } from "../components/Layout"
+import { Button } from "../components/Button"
+
+import LiteYouTubeEmbed from "react-lite-youtube-embed"
+import "react-lite-youtube-embed/dist/LiteYouTubeEmbed.css"
+
+import { lectures } from "../data"
+
+const ClientSidePdfDoc = React.lazy(() => import("../components/PdfDoc"))
+
+const Page: React.FC<PageProps> = (props) => {
+  const { t } = useTranslation()
+
+  const [activeLectureIndex, setActiveLectureIndex] = React.useState(0)
+  const [graypaperVisible, setGraypaperVisible] = React.useState(false)
+  const activeLecture = lectures[activeLectureIndex]
```

Comment:
Would be nice if this can also be passed in as a query parameter so people could share the link to the graypaper website directly instead passing the link to youtube of a specific lecture.

Something like this or whatever makes sense as a query parameter to select the section. Could be nice if the name of the section is also in the url 🤔 
```
graypaper.com/lectures#section=3
```

Reply:
I went with a client side implementation reading from pathname. The option with static page generation would also be possible if needed.